src/HealthChecks.AzureServiceBus/AzureServiceBusSubscriptionHealthCheck.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.ServiceBus.Management;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+
+namespace HealthChecks.AzureServiceBus
+{
+    public class AzureServiceBusSubscriptionHealthCheck : IHealthCheck
+    {
+        private static readonly ConcurrentDictionary<string, ManagementClient> ManagementClientConnections =
```

Comment:
Does `ManagementClient` implement `IDisposable`?

Reply:
No, it doesn't.